hooks/playbooks/control_plane_ironic.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Kustomize ControlPlane for ironic service
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Load vars from previous hooks
```

Comment:
this task is not need for ironic, only for jobs that use ceph.

Reply:
done